cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -680,7 +680,7 @@
 
   <property>
     <name>explore.executor.container.memory.mb</name>
-    <value>1024</value>
+    <value>${master.service.memory.mb}</value>
```

Comment:
This is explicitly set higher, by default, on purpose. Explore requires more memory to run than the Master service.